client.go

```diff
@@ -220,14 +225,15 @@ func (r *marathonClient) apiDelete(uri string, post, result interface{}) error {
 	return r.apiCall("DELETE", uri, post, result)
 }
 
+
 func (r *marathonClient) apiCall(method, uri string, body, result interface{}) error {
 	// Get a member from the cluster
 	marathon, err := r.cluster.GetMember()
 	if err != nil {
 		return err
 	}
 
-	url := fmt.Sprintf("%s/%s", marathon, uri)
+	url := fmt.Sprintf("%s/%s", marathon + "/marathon", uri)
```

Comment:
Where does this endpoint come from? Is that possibly DCOS-specific? Whatever it does, it's breaking the tests right now.